detox/test/e2e/03.actions-scroll.test.js

```diff
@@ -114,22 +112,6 @@ describe('Actions - Scroll', () => {
     await expect(element(by.text('HText8'))).not.toBeVisible();
   });
 
-  it(':android: should be able to scrollToIndex on vertical scrollviews', async () => {
-    // should ignore out of bounds children
-    await element(by.id('ScrollView161')).scrollToIndex(3000);
-    await element(by.id('ScrollView161')).scrollToIndex(-1);
-    await expect(element(by.text('Text1'))).toBeVisible();
-
-    await element(by.id('ScrollView161')).scrollToIndex(11);
-    await expect(element(by.text('Text12'))).toBeVisible();
-
-    await element(by.id('ScrollView161')).scrollToIndex(0);
-    await expect(element(by.text('Text1'))).toBeVisible();
-
-    await element(by.id('ScrollView161')).scrollToIndex(7);
-    await expect(element(by.text('Text8'))).toBeVisible();
-  });
-
```

Comment:
Why?

Reply:
It never worked. I'm deprecating this api

Reply:
Deprecating or removing?
WDYM never worked? We have the test right here 👆🏼

Reply:
Discussed F2F. This API is really not working. The test was a false positive 🤦🏼‍♂️ 
Let's @deprecate and remove it on next major